make/clean.mk

```diff
@@ -13,6 +13,7 @@ clean-users:
 	$(Q)-oc delete usersignups --all --all-namespaces
 	$(Q)-oc delete spacerequests --all --all-namespaces
 	$(Q)-oc delete spaces --all --all-namespaces
+	$(Q)-oc delete spacebindingrequests --all --all-namespaces
```

Comment:
You are right , we don't need to delete `spacebindingrequests` , but we do need to delete `spacerequests` because the spaces created by those cannot be deleted without deleting their SR resource.

I'll remove this line.

Reply:
Removed the deletion of `spacebindingrequests` in https://github.com/codeready-toolchain/toolchain-e2e/pull/779/commits/9fda3e49f9f08e51f3a46a124885bbc34f2b4e63